server/services/database/documents.js

```diff
@@ -110,6 +110,20 @@ const getDocumentIdByTranslationId = translationId => {
     .then(result => result.rows[0].document_id);
 };
 
+const deleteDocument = documentId => {
+  const query = "DELETE FROM documents WHERE id =$1";
+  return pool
+    .query(query, [documentId])
+    .then(result => result.rows);
+};
+
+const checkUserIdFromDocument = userId => {
```

Comment:
Not sure if this is correct as the parameter is named userId but you used it as a document Id in the sql query.

Reply:
You are querying the documents table with `id=$1`. `$1` is therefore the `id` of the document in this query, but the parameter passed to the function (and the query) is called `userId`.

Reply:
thanks Luke

Reply:
it is done